internal/handlers/pg/msg_insert.go

```diff
@@ -59,14 +59,6 @@ func (h *Handler) MsgInsert(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		return nil, err
 	}
 
-	created, err := h.pgPool.EnsureTableExist(ctx, sp.db, sp.collection)
```

Comment:
That was the only place in the code where a non-existing collection was handled.